### DISCUSSION TITLE
Nazwa koła naukowego

### DISCUSSION THREAD

**Discussion by dejwodejo** [math-insight/math-insight#2] [General]
Siemka, miejsce zbiorcze na pomysły odnośnie nazwy koła naukowego, możecie wpisać dowolny swój pomysł lub poprzeć czyiś, albo poprzeć aktualną nazwę "Koło Naukowe Matematyki Stosowanej". Pamiętajcie o tym, żeby miało to związek z matematyką.

**Comment by dejwodejo** [comment]
może coś a'la Mathemagica albo Enigma?

**Comment by marcin2203** [comment]
BazaUZ

**Reply by dejwodejo** [reply]
Baza to że spoko XD?

**Comment by dejwodejo** [comment]
Lambda? można to powiązać i z matmą i logo łatwo ułożyć

**Reply by marcin2203** [reply]
Sigma Club jak już xD

**Comment by dejwodejo** [comment]


**Reply by dejwodejo** [reply]
nie sa az tak tragiczne jak poprzednie XD
